CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to MentorPaired
+
+Thank you for indicating interest in contributing to MentorPaired. Following these guidelines help to communicate that you respect the time of the developers managing and working on this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and also helping you finalise your pull requests
+
+### What do I do? How do I get started?
+
+Start at the top, follow the [README.md](/README.md) guide. Now if you've noticed a bug or have a question, [search the issue tracker](https://github.com/mentorpaired/core/issues) to see if someone else already created a ticket. If that's not been done yet, go ahead and [make one](https://github.com/mentorpaired/core/issues/new).
```

Comment:
`Start at the top...` --> wondering how this can be rephrased.

If that's not been done yet --> If that has not been done yet

Reply:
I could also just remove it and begin the sentence with 'follow the README'..